hk25-ShallowCirc/update_fork.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+module load git
+
+# Ensure you're on the main branch
+git checkout main
+
+# Fetch latest changes from upstream
+git fetch upstream
+
+# Merge upstream/main into your main branch
+git merge upstream/main
```

Comment:
I would generally recommend a rebase, but that's up to you ;)
```suggestion
git rebase upstream/main
```